tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyMain.java

```diff
@@ -3598,7 +3598,7 @@ private JSONObject toJSONObject() throws JSONException {
                 .put("contextPath", contextPath)
                 .put("dbopen", dbopen)
                 .put("dbused", dbused)
-                .put("guests", guests)
+                .put("observers", observers)
```

Comment:
this affects telemetry data.  Probably fine, especialyl since we're not in a release.